Presentation.Web/app/components/it-system/usage/tabs/it-system-usage-tab-org.view.html

```diff
@@ -1,16 +1,5 @@
 ﻿<div class="object-section-container">
     <h4>{{systemUsageName}}</h4>
-    <div class="panel panel-default">
-        <div class="row">
-            <div class="col-xs-12">
-                <label>Relevante organisationsenheder</label>
```

Comment:
@a-naskret You did it right by the design proposal but thinking of it, I will reintroduce that header for the org tree. It simply makes more sense when that is there -- good work